Cargo.toml

```diff
@@ -15,16 +15,14 @@ fxhash = "0.2"
 log = "0.4"
 num-traits = "0.2"
 spirv = { package = "spirv_headers", version = "1.4.2", optional = true }
-glsl = { version = "4.1", optional = true }
 pomelo = { version = "0.1.4", optional = true }
 thiserror = "1.0"
 serde = { version = "1.0", features = ["derive"], optional = true }
 petgraph = { version ="0.5", optional = true }
 
 [features]
 default = []
-glsl_preprocessor = ["glsl"]
-glsl-new = ["pomelo"]
+glsl = ["pomelo"]
```

Comment:
let's call this `glsl-in` to be consistent

Reply:
done